api/api.yaml

```diff
@@ -638,9 +674,30 @@ paths:
                   description:
                     URL of where a zip of the source files can be downloaded
                     (e.g. Google Cloud Storage URL).
+                  example: https://storage.googleapis.com/my-bucket/source-files/integration-v1.2.3.zip
                 sourceYaml:
                   type: string
                   description: A YAML string that defines the integrations.
+                  example: |
```

Comment:
No need to provider an example here, it's too long, also this example is incorrect.